packages/page-accounts/src/Accounts/index.tsx

```diff
@@ -191,7 +191,7 @@ function Overview ({ className = '', onStatusChange }: Props): React.ReactElemen
           icon='plus'
           isDisabled={isIpfs}
           label={t<string>('Add account')}
-          onClick={toggleCreate}
+          onClick={() => setIsCreateOpen(true)}
```

Comment:
Currently after opening create account modal the browsers focus stays on open modal button. When you hit spacebar or enter key the modal will close. This will solve this issue.